rust/ql/lib/codeql/rust/internal/TypeInference.qll

```diff
@@ -578,14 +578,9 @@ private module CallExprBaseMatchingInput implements MatchingInputSig {
     }
 
     Declaration getTarget() {
-      result =
-        [
-          CallExprImpl::getResolvedFunction(this).(AstNode),
```

Comment:
Note, this line calls `getResolvedFunction` and casts to `AstNode`. The next line calls `getResolvedFunction` and calls to `Struct`. Seems like the later will always be included in the former.